build/media_source/plg_system_shortcut/js/keyselect.es6.js

```diff
@@ -0,0 +1,89 @@
+class JoomlaShortcutModal {
+	constructor() {
+		if (!Joomla) {
+			throw new Error('Joomla API is not properly initialised');
+		}
+		// Bindings
+		this.setModalAttributes = this.setModalAttributes.bind(this);
+		this.handleKeyCombinationkeyUpEvent = this.handleKeyCombinationkeyUpEvent.bind(this);
+		this.initialiseKeySelectModal = this.initialiseKeySelectModal.bind(this);
+		this.handleSaveCombinationkeyUpEvent = this.handleSaveCombinationkeyUpEvent.bind(this);
+		document.addEventListener('DOMContentLoaded',this.initialiseKeySelectModal);
+		let keySelectBtns = document.getElementsByClassName("keySelectBtn");
+		for(let x = 0; x < keySelectBtns.length; x+=1) {
+			keySelectBtns[x].addEventListener("click",this.handleKeySelectClickEvent,false);
+		}
+	}
+	handleKeyDownEvent(e) {
+		e.preventDefault();
```

Comment:
`	e.stopPropagation();
		e.stopImmediatePropagation();`


why use of these two ? read the issues while using